talk/expert/sfinae.tex

```diff
@@ -268,13 +268,13 @@
 \begin{frame}[fragile]
```

Comment:
I'm not allowed to comment above, but in line 252 and 258, `enable_if_t` is not c++11.